### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nechama-schles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1075,3 +1075,217 @@ Response:
...
+            "associatedSite": {
+                "id": "a54a5a61-2cc6-42cc-8468-8094d0595326",
+                "name": "ut",
+                "address": "185.229.31.138",
+                "resourceUri": "/disaster-recovery/v1beta1/virtual-sites/a54a5a61-2cc6-42cc-8468-8094d0595326"
+            },
+            "associatedVra": {
+                "name": "Z-VRA-soluta",
+                "ipAddress": "86.132.122.254",
+                "version": "8.0",
```
Comment (rosenfeld-hpe) [inline README.md:1104]: Change the version to have 3 octets, with double digits on the last one, like 8.0.10

File: README.md [hunk 2]
```diff
@@ -1075,3 +1075,217 @@ Response:
...
+                "status": "READY"
+            },
+            "vraConfiguration": {
+                "network": "37e1c5a5-2f9f-3be6-b786-dab1885ed62f",
+                "datastore": "52353a60-1b30-3a8a-9f4b-f6faa60333d2",
+                "setHostPassword": false,
+                "ipType": "DHCP",
```
Comment (rosenfeld-hpe) [inline README.md:1111]: Please change the IP type to be Static, I don't know how long we'll support DHCP in general
Also fill in valid ip addresses and subnet mask, etc.

File: README.md [hunk 3]
```diff
@@ -1075,3 +1075,217 @@ Response:
...
+    "associatedVma": {
+            "id":"UUID",
+            "name": "string",
+            "address": "string"
+            },
+    "defaultConfiguration": {
+        "network": "uuid",
+        "datastore": "uuid",
+        "setHostPassword": "bool",
+        "ipType": "string", Statsic or DHCP
```
Comment (rosenfeld-hpe) [inline README.md:1148]: Static, also remove DHCP, we are preventing deploy cluster from using DHCP

File: README.md [hunk 4]
```diff
@@ -1075,3 +1075,217 @@ Response:
...
+#### Deploy VRA (Host)
+POST /disaster-recovery/v1beta1/virtual-continuous-host-protectors/{hostID}/deploy
+
+Body: 
+```
+{
+"network": "c65d935a-9a6c-335e-a01a-48f55012701e",
+"datastore": "1a789dda-3b60-307d-8f77-e3c3d9ab28cd",
+"setHostPassword": true,
+"hostRootPassword": "QAD@ta2023!#", --> not mandatory and depand on the set above
```
Comment (rosenfeld-hpe) [inline README.md:1167]: Fix the typo in depand

File: README.md [hunk 5]
```diff
@@ -1075,3 +1075,217 @@ Response:
...
+POST /disaster-recovery/v1beta1/virtual-continuous-cluster-protectors/{clusterID}/deploy
+
+Body:
+```
+{
+    "network": "uuid",
+    "datastore": "uuid",
+    "setHostPassword": "bool",
+    "hostRootPassword": "string", 
+    "ipType": "string",--> Statsic or DHCP
```
Comment (rosenfeld-hpe) [inline README.md:1191]: Only Static

File: README.md [hunk 6]
```diff
@@ -1075,3 +1075,217 @@ Response:
...
+
+#### Edit VRA (Host)
+PUT /disaster-recovery/v1beta1/virtual-continuous-host-protectors/{hostID}
+
+Body: 
+```
+{
+    "setHostPassword": "bool",
+    "hostRootPassword": "string",
+    "ipType": "string",--> Statsic or DHCP
```
Comment (rosenfeld-hpe) [inline README.md:1214]: Static

File: README.md [hunk 7]
```diff
@@ -1075,3 +1075,217 @@ Response:
...
+
+Response: 
+```
+{
+    "taskUri": "/api/v1/tasks/e63b3f81-5ada-40b5-82ad-420af30715ed"
+}
+```
+
+#### Edit VRA (cluster)
+PUT /disaster-recovery/v1beta1/virtual-continuous-cluster-protectors/{clusterID}
```
Comment (rosenfeld-hpe) [inline README.md:1230]: We don't currently have Edit Cluster, and I don't know when we'll do
Anyway, there's the typo in Statsic

Comment (Nechama-gldr) [reply README.md:1230]: Lior added this API to the story.

File: README.md (hunk 1 above)
Comment (Nechama-gldr) [reply README.md:1104]: fixed

File: README.md (hunk 2 above)
Comment (Nechama-gldr) [reply README.md]: fixed

**Pull request merged**